### PR TITLE
[docs-only] Add WOPI image to dev docs

### PR DESCRIPTION
**Pull request by mmattel**
Fixes: #5329 (WOPI detailed image for dev-docs)

Adds a WOPI overview image to dev docs to explain how things are related together.

### CODE REVIEW

File: docs/ocis/deployment/ocis_wopi.md [hunk 1]
```diff
@@ -26,6 +26,10 @@ There are three oCIS app driver containers that register Collabora and OnlyOffic
 
 The last four containers are the WOPI server, Collabora and OnlyOffice.
 
+## Overview Image
+
+{{< figure src="/ocis/deployment/ocis_and_wopi_drawio.svg" >}}
```
Comment (kobergj) [inline docs/ocis/deployment/ocis_wopi.md:31]: Is this correct? Shouldn't it be 
```suggestion
{{< figure src="/docs/ocis/deployment/ocis_and_wopi_drawio.svg" >}}
```

**Review by kobergj — CHANGES_REQUESTED** [review]

File: docs/ocis/deployment/ocis_wopi.md (hunk 1 above)
Comment (mmattel) [reply docs/ocis/deployment/ocis_wopi.md:31]: This is a very good question. I ran it locally with `make -C . docs-serve` and it worked - where the command was executed in the docs folder. Any hints and tips are welcomed

Comment (mmattel) [reply docs/ocis/deployment/ocis_wopi.md:31]: @kobergj @micbar 
After a discussion with @micbar, I did a test in the root folder of ocis with and without using `/docs` in the path running `make -C . docs-serve`.

Only when NOT using `/docs`, the image gets rendered. This means, images are relative to the `/docs` path.

Comment (kobergj) [reply docs/ocis/deployment/ocis_wopi.md:31]: Ok great :+1:

**Review by kobergj — APPROVED** [review]

**Pull request merged**